### PR TITLE
Resource charts

### PR DESCRIPTION
**Pull request by s3rek**
- Added stacked columns with hero cards resource count in deck

- After clicking any of the columns, charts show the distribution of the resource icons among the cards.

- needed to import new module _**drilldown.js**_ and it depends on a newer version of _**highcharts.js**_

- Fixed colours of columns for Resource charts and Faction charts

image
image

### CODE REVIEW

**Comment by itskando** [comment]
Yes!! Thank you : j

**Comment by zzorba** [comment]
Thanks for this, I like the idea of drilling down.  Is there any option to customize that Back button, or is it basically always like that?

**Comment by s3rek** [comment]
https://api.highcharts.com/highmaps/drilldown.drillUpButton.theme

Looking through the documentation I found that there are some options, but I don't know what would You like this button to look like and if these options would be enough.

**Comment by zzorba** [comment]
Seeing it in action in their demo, maybe just an 'X' since it kind of looks more like a close than a back (based on the transition).

**Comment by s3rek** [comment]
image

Can do.

File: src/AppBundle/Resources/views/layout.html.twig [hunk 1]
```diff
@@ -153,7 +153,8 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/marked/0.3.5/marked.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.textcomplete/0.2.2/jquery.textcomplete.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.10.6/moment.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/highcharts/4.1.7/highcharts.js"></script>
+    <script src="https://code.highcharts.com/highcharts.js"></script>
```
Comment (Kamalisk) [inline src/AppBundle/Resources/views/layout.html.twig:156]: This doesn't link to a specific version. Would prefer if it did so it doesn't randomly break if they update things.

**Pull request merged**